libnavigator/src/main/java/com/mapbox/navigation/navigator/MapboxNativeNavigator.kt

```diff
@@ -12,6 +12,9 @@ import com.mapbox.navigator.RouterResult
 import com.mapbox.navigator.VoiceInstruction
 import java.util.Date
 
+/**
+ * Provides API to work with native Navigator class.
```

Comment:
```suggestion
 * Provides API to work with native Navigator class. Exposed for internal usage only.
```

Reply:
Refs. https://github.com/mapbox/mapbox-navigation-android/issues/2417

Reply:
I think it would be very useful to cover Navigator with docs anyway, At least because native interface generated with params like var1, var2. So very difficult to understand methods goals/logic without docs, wdyt?